Gemfile

```diff
@@ -34,15 +35,17 @@ else
       ver
     end
   end
-  gem 'rails', git: "git://github.com/rails/rails.git", tag: "v#{version}"
+  gem 'rails', git: "https://github.com/rails/rails.git", tag: "v#{version}"
 end
 
 if ENV['AREL']
   gem 'arel', path: ENV['AREL']
 end
 
 group :tinytds do
-  if ENV['TINYTDS_SOURCE']
+  if RUBY_ENGINE == 'jruby'
+    gem 'sqljdbc4', git: 'https://github.com/iaddict/sqljdbc4-java.git'
```

Comment:
Will the adapter function if this gem isn't installed? If so can we remove this? If not then this is a runtime dependency of the java target adapter correct?